roles/pulp/templates/settings.py.j2

```diff
@@ -17,3 +17,4 @@ DATABASES = {
 AUTHENTICATION_BACKENDS=['pulpcore.app.authentication.PulpNoCreateRemoteUserBackend']
 REMOTE_USER_ENVIRON_NAME="HTTP_REMOTE_USER"
 REST_FRAMEWORK__DEFAULT_AUTHENTICATION_CLASSES=('rest_framework.authentication.SessionAuthentication', 'pulpcore.app.authentication.PulpRemoteUserAuthentication')
+ENABLE_ANALYTICS = {{ pulp_enable_analytics | default(false) | bool | ternary('True', 'False') }}
```

Comment:
I think this should be `ANALYTICS` - https://github.com/theforeman/puppet-pulpcore/blob/master/templates/settings.py.erb#L17